source/AndroidResolver/src/PlayServicesResolver.cs

```diff
@@ -2140,11 +2151,12 @@ public static void SettingsDialog()
                             repoPath = relativePath;
                         }
 
-                        // If "Export Gradle Project" setting is enabled, gradle project expects
-                        // absolute path.
-                        if (exportEnabled) {
+                        // If "Export Gradle Project" setting is enabled, use absolute path
+                        // unless user has actively opted to use relative path
+                        if (exportEnabled && !useRelativeRepoPath) {
                             repoUri = String.Format("\"{0}\"",
-                                                    Path.Combine(projectFileUri, repoPath));
+                                                    FileUtils.PosixPathSeparators(
+                                                      Path.Combine(projectFileUri, repoPath)));
                         } else {
                             repoUri = String.Format("(unityProjectPath + \"/{0}\")", repoPath);
```

Comment:
I did tried your change locally and I noticed that this will generate the following block in exported `build.gradle`

```
([rootProject] + (rootProject.subprojects as List)).each { project ->
        def unityProjectPath = $/file:////Users/myusername/Workspace/myprojectfolder/$.replace("\\", "/")
        maven {
            url "https://maven.google.com"
        }
        maven {
            url (unityProjectPath + "/Assets/GeneratedLocalRepo/Firebase/m2repository") // Assets/Firebase/Editor/AnalyticsDependencies.xml:18, Assets/Firebase/Editor/AppDependencies.xml:22
        }
```

Note that this is still an absolute path instead of a relative path because it combines `unityProjectPath` and the relative path. For my instance, that will be `/file:////Users/myusername/Workspace/myprojectfolder//Assets/GeneratedLocalRepo/Firebase/m2repository`

I think the right way to do is to add another String field to specify the relative path from the project root to your export folder.

For instance, given the following project structure:
- Project root: "/foo"
- Export Android project path: "/foo/bar/baz"
- A new setting `Export Project Relative Path`: "bar/baz"

That way, the code can export something like the following as a relative path:

```
([rootProject] + (rootProject.subprojects as List)).each { project ->
        def unityProjectPath = $/file:////Users/myusername/Workspace/myprojectfolder/$.replace("\\", "/")
        maven {
            url "https://maven.google.com"
        }
        maven {
            url ("../../Assets/GeneratedLocalRepo/Firebase/m2repository") // Assets/Firebase/Editor/AnalyticsDependencies.xml:18, Assets/Firebase/Editor/AppDependencies.xml:22
        }
```